c/expr/py_expr.h

```diff
@@ -34,8 +34,7 @@ DECLARE_FUNCTION(
 DECLARE_FUNCTION(
   expr_reduceop,
   "expr_reduceop(op, col, groupby)\n\n"
-  "Compute a reduction over the provided column, using the provided Groupby\n"
-  "object.\n",
+  "Compute number of rows in each group, using the provided Groupby object.\n",
```

Comment:
Looks like the documentation of `expr_reduceop` and `expr_count` got mixed up

Reply:
You're right. I've fixed this in one of the commits, not sure how it reverted back.